troubleshooting/troubleshooting-ios-cannot-load-iframe.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Cannot Load iframe on iOS devices when using Cordova 5.0.0 or later
```

Comment:
The spelling of the title is quite inconsistent here. Typically, titles use title-style capitalization - that is you need to capitalize the first and the last word and all major words between them. 

In this case, the title should be "Cannot Load iframe on iOS Devices When Using Cordova 5.0.0 or Later"

Note that we preserve the typical capitalization for iframe and iOS.